nextflow_schema.json

```diff
@@ -234,6 +234,26 @@
                 }
             }
         },
+        "cellrangerarc_options": {
+            "title": "Cellranger ARC Options",
+            "type": "object",
+            "description": "Params related to the Cellranger pipeline",
+            "default": "",
+            "properties": {
+                "cellrangerarc_index": {
+                    "type": "string",
+                    "description": "Specify a pre-calculated cellranger index. Readily prepared indexes can be obtained from the 10x Genomics website. "
```

Comment:
Could you please clarify: Would normal cellranger indexes work here, or does cellranger arc use specific ones?

Reply:
Yes it seems to be the case (see https://kb.10xgenomics.com/hc/en-us/articles/4408281606797-Are-the-references-interchangeable-between-pipelines-).

Reply:
If they are the same, does it make sense to have different input parameters? Is it a flavour choice so that it is possible to split/differentiate them in different sections ( e.g. 'cellrangerarc_options' ) in the schema?

It is fine if it is or not, just want to get the whole idea.

Reply:
I'm fine with either solution, but would be great to add the link you posted to the documentation :)